packages/account/build/constants.js

```diff
@@ -97,6 +98,12 @@ const MINIMIZERS = !IS_RELEASE
       ];
 
 const plugins = () => [
+    new DefinePlugin({
+        'process.env.CROWDIN_PROJECT_NAME': JSON.stringify(process.env.PROJECT_NAME),
+        'process.env.CROWDIN_BRANCH_NAME': JSON.stringify(process.env.CROWDIN_BRANCH_NAME),
+        'process.env.ACC_TRANSLATION_PATH': JSON.stringify('deriv-app-account/staging'),
+        'process.env.CROWDIN_URL': JSON.stringify('https://translations.deriv.com'),
+    }),
     new CleanWebpackPlugin(),
```

Comment:
Since we are using @deriv-com/translation lib only in accounts package, will this have any impact on shared components ?